common/model/transaction.c

```diff
@@ -421,6 +421,29 @@ void transaction_reset(iota_transaction_t transaction) {
   memset(transaction->hash, FLEX_TRIT_NULL_VALUE, sizeof(transaction->hash));
 }
 
+uint16_t transaction_weight_magnitude(const iota_transaction_t transaction) {
```

Comment:
Hash is 243 long max, uint8_t is sufficient?

Reply:
.

Reply:
indeed